README.md

```diff
@@ -25,20 +25,23 @@ sudo apt update
 sudo apt install github-act-runner
 ```
 
-### configure the runner
+### add new runner
 ```console
-github-act-runner configure --url <github-repo-or-org-or-enterprise> --name <runner-name> -l <labels> --token <runner-registration-token>
+github-act-runner.sh new --owner <github-repo-or-org-or-enterprise> --name <runner-name> --labels <labels> --token <runner-registration-token>
```

Comment:
I usually see global scripts without extension. Do we need `.sh`?

Reply:
I think we don't, I thought about that it could be without suffix, but wasn't sure, feel free to remove it if you have no objections about that.

Reply:
Note, that there is also `restart.sh` script. In case you remove the suffix from it as well, don't forget to correct the `.service` file template as well inside the `github-act-runner` script.